.drone.yml

```diff
@@ -22,22 +22,20 @@ pipeline:
     commands:
       - tar -jxf owncloud-${SERVER_VERSION}.tar.bz2 -C /var/www
 
-  install-files-primary-s3:
-    image: owncloud/base:latest
-    pull: true
-    commands:
-      - cd /var/www/owncloud/apps
-      - git clone https://github.com/owncloud/files_primary_s3.git
-      - cd files_primary_s3
-      - wget https://raw.githubusercontent.com/composer/getcomposer.org/master/web/installer -O - -q | php -- --quiet
-      - php ./composer.phar install --no-dev --optimize-autoloader
-
   scality-waiting:
     image: owncloud/ubuntu:latest
     pull: true
     commands:
       - wait-for-it -t 300 scality:8000
 
+  create-bucket:
+    image: minio/mc
+    pull: true
+    environment:
+     - MC_HOSTS_scality=http://accessKey1:verySecretKey1@scality:8000
+    commands:
+     - mc mb scality/owncloud
```

Comment:
Does this enable versioning?

Reply:
Good point, I also completely forgot about the fact that `occ s3` does create a bucket and enables versioning..

Reply:
Hmm good catch - switched to aws-cli as poc here https://github.com/owncloud/smashbox-testing/pull/3